.github/fabricbot.json

```diff
@@ -2196,7 +2196,599 @@
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Hi @${issueAuthor}. We have added the \"s/try-latest-version\" label to this issue, which indicates that we'd like you to try and reproduce this issue on the latest available public version. This can happen because we think that this issue was fixed in a version that has just been released, or the information provided by you indicates that you might be working with an older version.\n\nYou can install the latest version by installing the latest Visual Studio (Preview) with the .NET MAUI workload installed. If the issue still persists, please let us know with any additional details and ideally a [reproduction project](https://github.com/dotnet/maui/blob/main/.github/repro.md) provided through a GitHub repository.\n\nThis issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time."
+            }
+          }
+        ]
+      }
+    },
+    {
+      "taskType": "scheduled",
+      "capabilityId": "ScheduledSearch",
+      "subCapability": "ScheduledSearch",
+      "version": "1.1",
+      "config": {
+        "frequency": [
+          {
+            "weekDay": 0,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 1,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 2,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 3,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 4,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 5,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          },
+          {
+            "weekDay": 6,
+            "hours": [
+              3,
+              15
+            ],
+            "timezoneOffset": 2
+          }
+        ],
+        "searchTerms": [
+          {
+            "name": "isOpen",
+            "parameters": {}
+          },
+          {
+            "name": "hasLabel",
+            "parameters": {
+              "label": "s/try-latest-version"
+            }
+          },
+          {
+            "name": "noActivitySince",
+            "parameters": {
+              "days": 7
+            }
+          }
+        ],
+        "taskName": "[Idle Issue Management] Close stale 's/try-latest-version' issues",
```

Comment:
Close the "please try latest version" labeled issues if no response comes within 7 days